cardinal/types/engine/info.go

```diff
@@ -0,0 +1,14 @@
+package engine
+
+type GetWorldResponse struct {
```

Comment:
Same feedback as above, the concept of "Response" struct type is something that is specific to server, you are not supposed to put in an omnibus types package.

Reply:
I'll see what I can do.

Reply:
Moved GetWorldResponse to server.